.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+#### Description of Change
+<!--
+Thank you for your Pull Request. Please provide a description above and review
+the requirements below.
+
+Contributors guide: https://github.com/TheAlgorithms/C-Plus-Plus/CONTRIBUTION.md
+-->
+
```

Comment:
@kvedala Consider adding a context section which you have mentioned in 'features requested', which can be required for pull request on non-issue subjects.

Reply:
please feel free to make a pull request with your suggestions to my fork. that way, it can be included.

Reply:
added in 800bd405cc9394368ff8963c2e33afac25dfef52